### PR TITLE
fix: Add manual-override "logout" workaround to both Portals

### PR DESCRIPTION
**Pull request by elwinschmitz**
[AB#30041](https://dev.azure.com/redcrossnl/7d17f2a9-3f18-40f1-8c29-98568fbca7e4/_workitems/edit/30041)

To make Portalicious deploy again, we need to also include the hotfixed-files...

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- ~[ ] I have added tests wherever relevant~
- [x] I have made sure that all automated checks pass before requesting a review
- [x] I do not need any deviation from our PR guidelines

### CODE REVIEW

File: interfaces/Portalicious/src/logout/index.html
Comment (aberonni) [inline interfaces/Portalicious/src/logout/index.html:1]: I don't understand why we need this on Portailcious?

File: interfaces/Portalicious/src/logout/logout.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+window.API_URL = `NG_URL_121_SERVICE_API`;
+
+function redirectToLogin() {
+  window.location.assign('/login?forced=' + Date.now());
```
Comment (aberonni) [inline interfaces/Portalicious/src/logout/logout.js:4]:
```suggestion
  window.location.assign('/login?cachebuster=' + Date.now());
```

Comment (elwinschmitz) [inline interfaces/Portalicious/src/logout/logout.js:4]:
```suggestion
  window.location.assign('/en/login?forced=' + Date.now());
```

File: interfaces/Portalicious/staticwebapp.config.json [hunk 2]
```diff
@@ -18,6 +24,10 @@
     {
       "route": "/nl/*",
       "rewrite": "/nl/index.html"
+    },
+    {
+      "route": "/*",
+      "redirect": "/en/index.html"
```
Comment (elwinschmitz) [inline interfaces/Portalicious/staticwebapp.config.json:30]:
```suggestion
```

**Review by elwinschmitz — COMMENTED** [review]
- Add E2E test (like `ChangePassword`) for LogOut
- Add documentation/explanation
- Fix suggestions

File: interfaces/Portalicious/src/logout/logout.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+  await window
+    .fetch(`${window.API_URL}/users/logout`, {
+      credentials: 'include',
+      method: 'POST',
+      mode: 'cors',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+        'X-121-Interface': 'portal', // See: services/121-service/src/shared/enum/interface-names.enum.ts
```
Comment (aberonni) [inline interfaces/Portalicious/src/logout/logout.js:23]: I would add a comment in `interface-enums` along the lines of

`This value is also hardcoded in interfaces/Portalicious/src/logout/logout.js - if you change it, update that too.`

File: interfaces/Portal/src/logout/README.md
Comment (aberonni) [inline interfaces/Portal/src/logout/README.md:1]: This file should be duplicated in the equivalent Portalicious folder

File: interfaces/Portal/src/logout/README.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# Manual Override Log-Out
+
+This folder contains the code for a "manual override" log-out or escape-hatch.
+This allows a user to log out of the platform manually, without a functioning Portal-application (due to a bug, out-of-date cache or some other browser-issue).
+
+The user would only have to visit the `/logout`-URL in their browser.
```
Comment (aberonni) [inline interfaces/Portal/src/logout/README.md:6]:
```suggestion
The user would only have to visit the `/logout`-URL in their browser.

**Do not move or change this folder unless you really know what you're doing.**
```

**Review by aberonni — CHANGES_REQUESTED** [review]

File: interfaces/Portal/src/logout/README.md
Comment (elwinschmitz) [reply interfaces/Portal/src/logout/README.md:1]: Oh, LOL. I already though I was seeing double... 👀

**Review by aberonni — APPROVED** [review]

**Pull request merged**